test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/schema/SchemaFile.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.justice.services.test.utils.core.schema;
+
+import static uk.gov.justice.services.test.utils.core.schema.SchemaDuplicateHelper.computeSha256Checksum;
+import static uk.gov.justice.services.test.utils.core.schema.SchemaDuplicateHelper.isJAR;
+import static uk.gov.justice.services.test.utils.core.schema.SchemaDuplicateHelper.stripWhitespace;
+
+import java.io.File;
+
+public class SchemaFile {
+
+    private File containerFile = null;
+
+    private File file = null;
+
+    private String originalFileContents = null;
+
+    private String originalFileChecksum = null;
+
+    private String prettyPrintChecksum = null;
+
+    public SchemaFile(File containerFile, File file) {
+        this.containerFile = containerFile;
+        this.file = file;
+    }
+
+    public SchemaFile(File containerFile, File file, String originalFileContents) {
+        this(containerFile, file);
+        setOriginalFileContents(originalFileContents);
```

Comment:
The file contents should not be set separately, or it risks inconsistent instance where the content is from a different file.